.github/workflows/build-test.yml

```diff
@@ -8,39 +8,39 @@ jobs:
     strategy:
```

Comment:
This file might benefit from updating via https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/ cookie-cutter.